KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutViewController.swift

```diff
@@ -305,13 +310,20 @@ final class KarhooCheckoutViewController: UIViewController, CheckoutView {
                                  paddingRight: standardSpacing)
         loyaltyView.anchor(top: rideInfoStackView.bottomAnchor, leading: rideInfoStackView.leadingAnchor, trailing: rideInfoStackView.trailingAnchor, paddingTop: standardPadding)
         
-        passengerDetailsAndPaymentView.anchor(top: loyaltyView.bottomAnchor,
+//        passengerDetailsAndPaymentView.anchor(top: loyaltyView.bottomAnchor,
```

Comment:
Some cleanuo and formatting should be done here. Can be postponed to the cleanup PR

Reply:
removed. Rest will be cleaned up in next ticket.